services/src/bin/main.rs

```diff
@@ -6,9 +6,14 @@ use geoengine_services::workflows::registry::HashMapRegistry;
 use geoengine_services::handlers;
 use geoengine_services::users::userdb::HashMapUserDB;
 use geoengine_services::handlers::handle_rejection;
+use geoengine_common::config::CONFIG;
+use std::error::Error;
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<(), Box<dyn Error>> {
+    let host: std::net::IpAddr = CONFIG.read()?.get::<String>("web.host")?.parse()?;
```

Comment:
Is there a way to use symbols instead of strings? This could be useful for refactoring.

cf. https://github.com/mehcode/config-rs/blob/master/examples/hierarchical-env/src/settings.rs

Reply:
like a constant instead of "web.host"? In the link you posted they use hard-coded strings too...
Can you give an example?

Reply:
https://github.com/geo-engine/rfcs/issues/7

Reply:
https://docs.rs/arc-swap/0.4.7/arc_swap/